core/src/main/scala/scalacache/package.scala

```diff
@@ -126,7 +128,7 @@ package object scalacache extends StrictLogging {
    *
    * @tparam V the type of values that the cache will accept
    */
-  def typed[V](implicit scalaCache: ScalaCache) = new TypedApi[V]()(scalaCache)
+  def typed[V: Codec](implicit scalaCache: ScalaCache) = new TypedApi[V]()(scalaCache, implicitly[Codec[V]])
```

Comment:
Please add a parameter to the implicit param list instead of using implicitly.

(See slide 84 of http://plastic-idolatry.com/erik/sw2015.pdf)